api/resources_portal/views/material_request.py

```diff
@@ -150,12 +152,15 @@ def update(self, request, *args, **kwargs):
 
         if request.user == material_request.requester:
             if "irb_attachment" in request.data:
-                material_request.irb_attachment = serializer.validated_data["irb_attachment"]
+                irb = serializer.validated_data["irb_attachment"]
+                material_request.irb_attachment = irb
+                # allow the assignee read access to the attachment
+                assign_perm("view_attachment", material_request.assigned_to, irb)
```

Comment:
Limiting this to just the assignee seems overly restrictive I think. I'm not actually sure though so I'm gonna loop in @dvenprasad: Are IRBs so private that it'd be a problem to let anyone who can manage requests see them?

Reply:
I think one potential solution to this could be to make `view_attachments` a permission for organizations, then once we make those granular enough owners could restrict which members can see attachments if they care.

Reply:
Ok so Deepa and I talked. My last suggestion was even too granular. If someone can review a request for an organization, they should be able to view its attachments. Once again, `assigned_to` is just to know where the buck stops, not for permissions.